README.md

```diff
@@ -25,7 +24,7 @@ Same as in Node.js, you just have to [browserify](https://github.com/substack/js
 
 Make the [multihashes.min.js](/dist/multihashes.min.js) available through your server and load it using a normal `<script>` tag, this will export the `multihashes` constructor on the `window` object, such that:
 
-```JavaScript
+```javascript
```

Comment:
True, it will work, even with the weird capitalization, but it doesn't match the other code blocks in the document, so I normalized the whole thing with [tidy-markdown](https://www.npmjs.com/package/tidy-markdown).